### ISSUE TITLE
Failed to build tarballs

### ISSUE THREAD

**Issue by haricnugraha** [hyperjumptech/monika#1313]
**Describe the bug**
Failed to build tarballs. More info https://github.com/hyperjumptech/monika/actions/runs/10003848111/job/27651660920.

**To Reproduce**

Run `npm run pack-tarballs`.

**Expected behavior**
Tarballs built successfully.

**Screenshots**
<img width="1117" alt="Screenshot 2024-07-19 at 1 57 18 PM" src="https://github.com/user-attachments/assets/40196564-a3bd-46d3-a971-828f5668d338">

**Desktop (please complete the following information):**

- OS: Ubuntu
- Version: 20

**Comment by sapiderman** [comment]
ooooh deja vu.  I think becuase we update to new oclif version
we've had some p*cking problems before.

poke: @suyono3484  @syamsudotdev #1255 #1311

**Comment by suyono3484** [comment]
Confirmed. I can reproduce the problem.
![image](https://github.com/user-attachments/assets/80801cf5-c627-449f-b10d-38ad88dfd113)


Looking at the error message, the command `npm run pack-tarballs` doesn't trigger `tsc -b`, hence missing all the `.js` files.
I tried to run `npm run prepack` before issuing `npm run pack-tarballs` and it worked.
![image](https://github.com/user-attachments/assets/4b5fd4ea-8dc5-4ba6-85d2-c76906402c95)


Should we update the workflow? @haricnugraha
